paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -159,6 +159,7 @@ private fun SemanticsNode.accessibilityText(): String? {
   val onClickLabel =
     if (disabled != null) null else config.getOrNull(SemanticsActions.OnClick)?.label?.let { "<on-click>: $it" }
   val heading = if (config.getOrNull(SemanticsProperties.Heading) != null) "<heading>" else null
+  val toggleableState = config.getOrNull(SemanticsProperties.ToggleableState)?.let { "<toggleable: ${it.name}>" }
```

Comment:
```suggestion
  val toggleableState = config.getOrNull(SemanticsProperties.ToggleableState)?.let { "<toggleable>: ${it.name}" }
```

nit: i'm going solely by the onClick implementation, feel free to ignore if you have reasons